qos/cosmos/qos.go

```diff
@@ -54,17 +55,24 @@ func NewQoSInstance(logger polylog.Logger, config CosmosSDKServiceQoSConfig) *Qo
 		endpointStore:    store,
 	}
 
-	cosmosSDKRequestValidator := &cosmosSDKRequestValidator{
+	requestValidator := &requestValidator{
 		logger:       logger,
 		serviceID:    serviceId,
 		chainID:      cosmosSDKChainID,
 		serviceState: serviceState,
```

Comment:
The request validator does not contain the required `supportedAPIs` map, which causes requests to fail in local testing.

I've opened a small PR to fix this and to align the language of what was previously called `rpcTypes` to the terminology used in this PR (`supportedAPIs`): https://github.com/buildwithgrove/path/pull/365